src/styles/abstract/_mixins.scss

```diff
@@ -0,0 +1,11 @@
+@mixin font($size, $weight: 400, $height: normal, $style: normal) {
+  font-size: $size;
+  line-height: $height;
+  font-weight: $weight;
+  font-style: $style;
+}
+
+@mixin border($size, $color, $raduis: none) {
+  border: $size solid $color;
+  border-radius: $raduis;
```

Comment:
Typo in the mixin argument: 'raduis' should be 'radius'. This is important for code readability and to avoid confusion.